Lib/test/test_concurrent_futures/test_wait.py

```diff
@@ -67,9 +84,14 @@ def test_first_exception(self):
         self.assertEqual(set([future1, future2]), finished)
         self.assertEqual(set([future3]), pending)
 
+        t.join()
```

Comment:
Use maybe `threading_helper.join_thread(t)` (from test.support.threading_helper) to not wait forever if something goes wrong.